crates/holochain_integrity_types/src/link.rs

```diff
@@ -194,3 +184,40 @@ impl TryInto<String> for LinkTag {
         String::from_utf8(self.0)
     }
 }
+
+/// convert `LinkTag` into `SerializedBytes` (Infallible)
+impl From<LinkTag> for SerializedBytes {
+    fn from(tag: LinkTag) -> SerializedBytes {
+        SerializedBytes::from(UnsafeBytes::from(tag.0))
+    }
+}
+
+/// Creates a `LinkTag` from `SerializedBytes` (Infallible)
+impl From<SerializedBytes> for LinkTag {
+    fn from(sb: SerializedBytes) -> Self {
+        Self::new(sb.bytes().clone())
+    }
+}
+
+#[cfg(test)]
+mod test {
+    use super::*;
+
+    #[derive(Clone, Debug, Serialize, Deserialize, SerializedBytes)]
+    pub struct Data {
+        pub latitude: f64,
+        pub longitude: f64,
+    }
+    #[test]
+    fn link_tag_roundtrip() {
+        let location = Data {
+            latitude: 4.518758758758,
+            longitude: 4.718758758973,
+        };
+        let sb = SerializedBytes::try_from(location.clone()).unwrap();
+        let tag = LinkTag::from(sb);
+        let back_to_sb: SerializedBytes = tag.into();
+        let back_to_location: Data = back_to_sb.try_into().unwrap();
+        assert_eq!(location.latitude, back_to_location.latitude);
```

Comment:
These objects should be equal

```suggestion
        assert_eq!(location, back_to_location);
```

and derive `PartialEq` on the `Data` type?